src/parser/shaman/elemental/modules/talents/Ascendance.js

```diff
@@ -35,36 +52,94 @@ class Ascendance extends Analyzer {
     return this.rawUpdate;
```

Comment:
These two getters are essentially the same and `rawUpdate` isn't used anywhere else
```suggestion
    return this.selectedCombatant.getBuffUptime(SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id) / this.owner.fightDuration;
```